bergen.py

```diff
@@ -19,8 +19,11 @@ def main(config):
 
     if 'train' in config:
         rag.train()
-
-    rag.eval(dataset_split='dev')
+    if 'dataset_split' in config:
```

Comment:
What does this have to do with LLM-eval?

Reply:
you can perform eval on train.  I do it for WTR.  you can alos do it on 'test' !